### PR TITLE
create new methods to handle special kind in proper way

### PR DESCRIPTION
**Pull request by KengoTODA**
To fix problem in current way, mark them as deprecated and create new methods.

refs #23

### CODE REVIEW

File: findbugs/src/java/edu/umd/cs/findbugs/OpcodeStack.java [hunk 1]
```diff
@@ -268,11 +272,25 @@
         public static final @SpecialKind
         int TYPE_ONLY = 24;
 
+        /**
+         * @deprecated will be deleted at v4 release. use @{link {@link #getSpecialKindName(int)}} instead.
```
Comment (jsotuyod) [inline findbugs/src/java/edu/umd/cs/findbugs/OpcodeStack.java]: double `@link`?

File: findbugs/src/java/edu/umd/cs/findbugs/OpcodeStack.java [hunk 2]
```diff
@@ -268,11 +272,25 @@
...
+         */
+        @Deprecated
         @edu.umd.cs.findbugs.internalAnnotations.StaticConstant
         public static final HashMap<Integer, String> specialKindNames = new HashMap<Integer, String>();
```
Comment (jsotuyod) [inline findbugs/src/java/edu/umd/cs/findbugs/OpcodeStack.java]: we should probably leave a `TODO` for ourselves to convert this into a `private ConcurrentMap`

Comment (KengoTODA) [inline findbugs/src/java/edu/umd/cs/findbugs/OpcodeStack.java]: Which we prefer: TODO comment or Issue at GitHub? Personally I prefer latter one.

File: findbugs/src/java/edu/umd/cs/findbugs/OpcodeStack.java (hunk 1 above)
Comment (KengoTODA) [inline findbugs/src/java/edu/umd/cs/findbugs/OpcodeStack.java]: Oops... I'll fix.

**Comment by jsotuyod** [comment]
Ok, Github issue it is. I just created a new 4.0.0 milestone so we can tag it.

**Comment by KengoTODA** [comment]
@jsotuyod I've issued #28, please tag it with new milestone.

**Comment by jsotuyod** [comment]
@KengoTODA done! thanks!

**Pull request merged**